File: solutions/3-consonants/solution.c
#include <stdio.h>
int isVowel[200], n, numVowels, numConsonants;
int main() {
	isVowel['A'] = isVowel['E'] = isVowel['I'] = isVowel['O'] = isVowel['U'] = 1;
	scanf("%d", &n);
	for (int i = 0; i < n; i++) {
		char c;
		scanf(" %c", &c);
		numVowels += isVowel[c];
		numConsonants += !isVowel[c];
	}
	if (numVowels > numConsonants) printf("VOWELS\n");
	else if (numVowels < numConsonants) printf("CONSONANTS\n");
	else printf("EQUAL\n");
}

File: solutions/3-consonants/solution.py
N = int(input())
vowels = 0
for char in input().strip():
	if char in 'AEIOU':
		vowels += 1
consonants = N - vowels
if vowels > consonants:
	print('VOWELS')
elif vowels < consonants:
	print('CONSONANTS')
else:
	print('EQUAL')